.changeset/afraid-cooks-shave.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ai-sdk/orchestration': minor
+---
+
+[Compatibility Note] Deprecate `buildAzureContentFilter()` function since it restricts filtering to have only one filter. 
+Use `ContentFilters.azureContentSafety()` instead.
```

Comment:
[minor] The format of the input also changed, the user has to call the function and explicitly pass it into an array now within a filter object. That there is a behaviour change is not very evident from the release note. Maybe linking to code samples helps?

Reply:
As discussed, please follow the verb styleGuide for the function naming.

Reply:
As requested, it is called `ContentFilters.buildAzureContentSafety()` or `buildAzureContentSafety()` now. Left some comments in the BLI though.